src/lib/types/minions.ts

```diff
@@ -346,6 +346,11 @@ export interface GuardiansOfTheRiftActivityTaskOptions extends MinigameActivityT
 	combinationRunes: boolean;
 }
 
+export interface NightmareZoneActivityTaskOptions extends MinigameActivityTaskOptions {
+	strategy: string;
```

Comment:
strategy should be an enum, using a single source/enum for anywhere the strategy is used (e.g. autocomplete)